Dockerfile

```diff
@@ -11,6 +11,10 @@ ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
+ARG MESOS_RELEASE
+ARG STORM_RELEASE
+ARG MIRROR
```

Comment:
There are default values set in bin/build-release.sh (since the environment variables are always set to something, so I think it would just be making those values redundantly specified unless I'm missing something?